tests/test_ax_generators.py

```diff
@@ -74,9 +76,14 @@ def test_ax_single_fidelity():
     var1 = VaryingParameter("x0", -50.0, 5.0)
     var2 = VaryingParameter("x1", -5.0, 15.0)
     obj = Objective("f", minimize=False)
+    p1 = Parameter("p1")
 
     gen = AxSingleFidelityGenerator(
```

Comment:
Constraint parameter capabilities were also added to `AxMultiFidelityGenerator`.
Don't you want to add a test for this too?

Reply:
Thanks for the suggestion. Apparently the multi-fidelity generator does not accept `parameter_constraints` (see https://github.com/facebook/Ax/issues/1639), so I removed them. I updated the tests to include `outcome_constraints`.

Reply:
Alright! Thanks!